feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html

```diff
@@ -43,7 +43,10 @@
     *ngIf="customerSupportAgentLoggedIn$ | async; else showLoginForm"
   >
     <ng-container *ngIf="customer$ | async; else showCustomerSelection">
-      <cx-customer-emulation *ngIf="notCollapsed"></cx-customer-emulation>
+      <cx-customer-emulation
+        *ngIf="notCollapsed"
+        (submitEvent)="startCustomerEmulationSession($event)"
```

Comment:
Why is `(submitEvent)="startCustomerEmulationSession($event)"` added and which component is listening for it?

Reply:
The asm-main component is listening for when the cart is assigned to the selected customer so it can refresh the customer session with the selected cart.